src/App.js

```diff
@@ -42,21 +36,6 @@ import Logo from './film-indy-logo.png'
 
 import * as accountActions from './redux/actions/creators/accountActions'
 
-const ALGOLIA_SEARCH_KEY = process.env.REACT_APP_ALGOLIA_SEARCH_KEY
-const ALGOLIA_APP_ID = process.env.REACT_APP_ALGOLIA_APP_ID
-
-
-const AutoCompleteBar = connectAutoComplete(
```

Comment:
That should still stay there since its the search bar for most of the site.
If you want to hide the search bar in the nav bar on the home page, thats doable but we shouldn't remove the entire thing. Just hide it for that page.

There should be a object called location in props which will give you a string which tells you what page you are on. You can some string matching and figure out if you want to display the search bar if you are not on the home page.

Reply:
That worked great, this branch should be set.